composer.json

```diff
@@ -14,20 +14,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "guzzlehttp/guzzle": "6.2.1",
+        "kevinrob/guzzle-cache-middleware": "1.4.1"
     },
     "require-dev": {
         "phpunit/phpunit": "4.8.26",
         "phpdocumentor/phpdocumentor": "2.*",
         "predis/predis": "1.0.*",
-        "guzzlehttp/guzzle": "5.*",
-        "guzzlehttp/cache-subscriber": "0.1.*",
```

Comment:
This goes away because it is not compatible with Guzzle 6: https://github.com/guzzle/cache-subscriber